### PR TITLE
Merge develop 8.0 30.06.2016

### PR DESCRIPTION
**Pull request by livecodepanos**
Clean merge + version update

### CODE REVIEW

File: version [hunk 1]
```diff
@@ -1,6 +1,6 @@
-BUILD_REVISION = 13022
+BUILD_REVISION = 13023
 BUILD_MAJOR_VERSION = 8
 BUILD_MINOR_VERSION = 0
-BUILD_POINT_VERSION = 2
```
Comment (peter-b) [inline version:4]: I think this should still be 2.

**Comment by peter-b** [comment]
@livecode-vulcan review ok 1132805

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok 1132805c1854697c39f0e3f07703ab6c57f50bc3

**Comment by livecode-vulcan** [comment]
😎 test success 1132805c1854697c39f0e3f07703ab6c57f50bc3
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/2725)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/2735)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/2720)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/2736)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/2742)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/2748)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/2760)

**Pull request merged**